packages/blocks/src/store/selectors.js

```diff
@@ -237,10 +237,17 @@ export const getBlockVariations = createSelector(
 export function getActiveBlockVariation( state, blockName, attributes, scope ) {
 	const variations = getBlockVariations( state, blockName, scope );
 
-	const match = variations?.find( ( variation ) => {
+	if ( ! variations ) {
```

Comment:
I wasn't sure about this like but `getBlockVariations` returns a non-empty array or undefined after converting the array into the return value.